src/scripts/app.jsx

```diff
@@ -12,20 +12,28 @@ import "../styles/main.sass";
 const auth = resoundAPI.auth;
 
 class Root extends React.Component {
+  redirectIfLoggedOut() {
+    if (!auth.isAuthenticated()) {
```

Comment:
Just a stylistic thing that's optional... for functions that take no arguments and behave as "computed" properties, I like to make them getters.  So in `src/scripts/services/auth.js`, `isAuthenticated(){...}` would be `get isAuthenticated(){...}`.  That way you don't have to provide empty parenthesis.  There's no objective reason why you'd have to do this, but thought I'd throw that out there in case you didn't know you could do that.